packages/app/src/specs/banners/TrackedBanner.vue

```diff
@@ -8,9 +8,36 @@
   </Alert>
 </template>
 
+<script lang="ts">
+import { CohortConfig, useCohorts } from '@packages/frontend-shared/src/composables/useCohorts'
+
+export type CohortOption = {
+  cohort: string
+  value: string
+}
+
+export const getOptionForCohort = (name: string, options: CohortOption[]) => {
+  const cohortConfig: CohortConfig = {
+    name,
+    cohorts: options.map((option) => option.cohort),
+  }
+
+  const cohortSelected = useCohorts(cohortConfig)
+
+  return computed(() => {
+    if (cohortSelected.value) {
+      return options.filter((option) => option.cohort === cohortSelected.value)[0]
```

Comment:
`options.find((option) => option.cohort === cohortSelected.value)`

Reply:
I knew there was something that would get the first value out of the array but `find` was not coming to mind. Thanks!